src/openzaak/components/zaken/api/urls.py

```diff
@@ -55,6 +56,8 @@
     basename="zaakidentificatie_alias",
 )
 router.register("zaak_registreren", ZaakRegistrerenViewset, basename="registreerzaak")
+zaakopschorten_view = ZaakOpschortenViewset.as_view({"post": "post"})
```

Comment:
to have the url POST `/api/v1/zaak_opschorten/<zaak_uuid>` i had to create a custom method and map it to POST, which meant i could not get it working with the router, it works as a separate path but stuff like the default error example responses are not added in the api spec.

other solutions would be to change it to PUT/PATCH, do something with the router (SimpleRouter.routes) or to add it as an action of zaak (`/api/v1/zaken/<zaak_uuid>/zaak_opschorten`) but then the other conv endpoints should maybe also become actions of their main object?